### PR TITLE
W1 D2 curriculum specialist updates

### PR DESCRIPTION
**Pull request by nsea-log**

### CODE REVIEW

File: tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial4.ipynb
Comment (iamzoltan) [inline tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial4.ipynb:814]: We need to change this back to python3 before pushing

**Review by iamzoltan — CHANGES_REQUESTED** [review]
Things look good. We need to make sure the kernel is set to the standard python3 for the build to work before we can merge.

File: tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial4.ipynb
Comment (nsea-log) [reply tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial4.ipynb]: True, sorry for that. Pushed update commit.

File: tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial1.ipynb
Comment (iamzoltan) [inline tutorials/W2D4_ClimateResponse-Extremes&Variability/W2D4_Tutorial1.ipynb:770]: Look like you accidentally added this kernel change to this PR.

**Review by iamzoltan — COMMENTED** [review]
Looks good. One minor change, and we should be good to go.

File: tutorials/W1D2_StateoftheClimateOceanandAtmosphereReanalysis/W1D2_Tutorial3.ipynb
Comment (iamzoltan) [inline tutorials/W1D2_StateoftheClimateOceanandAtmosphereReanalysis/W1D2_Tutorial3.ipynb:525]: Do you want to add something before we merge the changes?

**Review by iamzoltan — COMMENTED** [review]
Looks good!

Did you intentionally include changes to W2D4?

File: tutorials/W1D2_StateoftheClimateOceanandAtmosphereReanalysis/W1D2_Tutorial3.ipynb
Comment (nsea-log) [reply tutorials/W1D2_StateoftheClimateOceanandAtmosphereReanalysis/W1D2_Tutorial3.ipynb:525]: Will remove this comment for now, will think about it and wait for feedback from the pre-pod.

Comment (nsea-log) [reply tutorials/W1D2_StateoftheClimateOceanandAtmosphereReanalysis/W1D2_Tutorial3.ipynb]: Most changes in W2D4 were made intentionally but on the wrong branch. Hope that works out... Sorry, my bad.

**Pull request merged**
